addon/mixins/child.js

```diff
@@ -55,11 +55,18 @@ export default Mixin.create({
     }
   },
 
+  notifyParentOfInsert: on('didInsertElement', function() {
```

Comment:
We should use `didInsertParent` instead of `on('didInsertParent',`. It makes the calling order more predictable and provides a better opportunity for subclasses to override the method.